src/pages/clients/clients.jsx

```diff
@@ -17,14 +21,51 @@ export default function ClientsPage() {
 	const DynamicWrapper = dynamic(() => import('../../components/styledClientWrapper'), {
 		ssr: false,
 	});
+	const [inputValue, setInputValue] = useState('');
+
+	const fuzzyResult = search(inputValue, clients, {
+		keySelector: obj => obj.CompanyName,
```

Comment:
Better write object keys in `camelCase`   🤓

```suggestion
		keySelector: obj => obj.companyName,
```